modules/gapi/test/common/gapi_render_tests.hpp

```diff
@@ -52,18 +82,35 @@
 #define text1 Prim{cv::gapi::wip::draw::Text{"TheBrownFoxJump", cv::Point{100, 100}, FONT_HERSHEY_SIMPLEX, 2, cv::Scalar{102, 178, 240}, 1, LINE_8, false} }
 #define texts Prims{text1}
 
-namespace opencv_test
-{
+        switch ((Value)val_)
+        {
+        case RECTS: return rects;
+        case CIRCLES: return circles;
+        case LINES: return lines;
+        case MOSAICS: return mosaics;
+        case IMAGES: return images;
+        case POLYGONS: return polygons;
+        case TEXTS: return texts;
+        }
+        CV_Error(Error::StsInternal, "");
+    }
+}; // PrimsTestParam
```

Comment:
What is this? Why do have a lots of `#define`s? why all those are defined in the header file? why all those are defined within a class definition??

Reply:
This is also a comment for @TolyaTalamanov  :)